e2/tsconfig.json

```diff
@@ -1,14 +1,10 @@
 {
   "extends": "@electron-toolkit/tsconfig/tsconfig.json",
   "include": [
-    "../app/**/*",
-    "../backend/**/*",
-    "src/renderer/src/env.d.ts",
-    "src/renderer/src/**/*",
-    "src/renderer/src/**/*.svelte",
-    "../electron/src/logic.ts",
-    "../electron/src/logic/**/*",
-    "src/preload/*.d.ts"
```

Comment:
Why these removals?

Reply:
Did you check whether it still works when the app is built as package/installer? (`yarn run build:linux && ./dist/*

Reply:
I didn't even mean to commit that change, but I wanted my `npx tsc --noEmit` to mostly work.

Reply:
Fair enough. I can remove the change before landing.